.github/workflows/presubmit.yml

```diff
@@ -28,17 +28,25 @@ jobs:
           submodules: recursive
       - name: Build
         run: |
-            mkdir build
-            cd build
+            if [[ "${{ matrix.os }}" == "ubuntu-20.04" ]]; then
+              sudo apt update
+              sudo apt install -y libtclap-dev libglm-dev libglew-dev libsfml-dev libstb-dev libidn11 libx11-dev libxrandr-dev libxi-dev mesa-common-dev libgl1-mesa-dev libglu1-mesa-dev libudev-dev
```

Comment:
I know Ubuntu isn't universal, but it might be helpful to put this in the readme.  Or at least the ones that aren't there by default on a desktop install.

I had to install these to get the SDK to build:

```sh
sudo apt install libtclap-dev libstb-dev libglew-dev libsfml-dev libglm-dev
```

Reply:
These packages are mentioned in the OpenCL-Guide in the Linux guide [here](https://github.com/KhronosGroup/OpenCL-Guide/pull/17/files#diff-89767f088390b2e71cfc4717bbfe10a989e2d42346d89e34e5b76510992845b7R124). These are only required if one builds the samples and not otherwise.

Reply:
I'm guessing this is only needed if the package dependencies aren't satisfied via vcpkg?

Regardless, I agree it would be worth including this as an alternative in the README.  Installing via the OS package manager will likely be smoother than vcpkg (getting it working on Ubuntu wasn't trivial) and this is the mechanism we're testing in CI.

Reply:
One can install them either way. It's user preference.